src/main/java/net/dv8tion/jda/api/entities/GuildScheduledEvent.java

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.dv8tion.jda.api.entities;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.managers.GuildScheduledEventManager;
+import net.dv8tion.jda.api.requests.RestAction;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+/**
+ * A class representing a guild scheduled event (the ones that show up under the events tab in the Official
+ * Discord Client). These events should not be confused with {@link net.dv8tion.jda.api.events Gateway Events}, which
+ * are fired by Discord whenever something interesting happens
+ * (ie., a {@link net.dv8tion.jda.api.events.message.MessageDeleteEvent MessageDeleteEvent} gets fired whenever a message gets deleted).
+ */
+public interface GuildScheduledEvent extends ISnowflake
+{
+    /**
+     * The maximum allowed length for an event's name.
+     */
+    int MAX_NAME_LENGTH = 100;
+
+    /**
+     * The maximum allowed length for an event's description.
+     */
+    int MAX_DESCRIPTION_LENGTH = 1000;
+
+    /**
+     * The name of the event.
+     *
+     * @return The name
+     */
+    @Nonnull
+    String getName();
+
+    /**
+     * The description of the event.
+     *
+     * @return The description, or {@code null} if none is specified
+     */
+    @Nullable
+    String getDescription();
+
+    /**
+     * The user who originally created the event.
+     *
+     * @return The user, or {@code null} if the event was created before Discord started keeping track of event
```

Comment:
Can you still get those events?

Reply:
The [scheduled event docs](https://discord.com/developers/docs/resources/guild-scheduled-event) imply that its possible:
> `creator_id` will be null and creator will not be included for events created before October 25th, 2021, when the concept of `creator_id` was introduced and tracked.